src/components/Scenario.vue

```diff
@@ -51,6 +51,22 @@
                 </select>
               </span>
             </div>
+            <div class="scenario-line" v-if="scenario.software.name">
+              <span class="scenario-line-caption">
+                <p>📌 Instances de {{ getSoftwareOption(scenario.software.name) }}</p>
+              </span>
+              <IncrementButton 
+                v-model="scenario.numberOfInstances"
```

Comment:
Plutôt le mettre dans le software ?
```suggestion
                v-model="scenario.software.numberOfInstances"
```

Reply:
Bonne idée ! Fixé dans 92a472e30f563dd3a8c196cf82e62ae8ec71a590. Est-ce que ça convient ?